server/data/mappers/actionPlanMapper.test.ts

```diff
@@ -47,4 +50,48 @@ describe('actionPlanMapper', () => {
     // Then
     expect(actionPlan).toEqual(expectedActionPlan)
   })
+
+  it('should map to ActionPlan given valid ActionPlanResponse containing several goals', () => {
```

Comment:
This is the new test that asserts that when an Action Plan is mapped via the mapper, the Goals are correctly mapped with the correct sequence number